src/core/scripts/app.js

```diff
@@ -4,7 +4,7 @@ angular.module('angularity', [
         'ngTouch',
         'ngRoute'
     ])
-    .config(function($httpProvider, $routeProvider, $locationProvider) {
+    .config(($httpProvider, $routeProvider, $locationProvider) => {
```

Comment:
Well, then I can also convert all the top level controllers that use injection to use @ng-inject as well!